src/compressor/CMakeLists.txt

```diff
@@ -11,15 +11,18 @@ set(compressor_plugin_dir ${CMAKE_INSTALL_PKGLIBDIR}/compressor)
 add_subdirectory(snappy)
 add_subdirectory(zlib)
 add_subdirectory(zstd)
+add_subdirectory(lz4)
 
 add_custom_target(compressor_plugins DEPENDS
     ceph_snappy
     ceph_zlib
-    ceph_zstd)
+    ceph_zstd
+    ceph_lz4)
 
 if(WITH_EMBEDDED)
   include(MergeStaticLibraries)
   add_library(cephd_compressor_base STATIC ${compressor_srcs})
   set_target_properties(cephd_compressor_base PROPERTIES COMPILE_DEFINITIONS BUILDING_FOR_EMBEDDED)
-  merge_static_libraries(cephd_compressor cephd_compressor_base cephd_compressor_snappy cephd_compressor_zlib cephd_compressor_zstd)
+  merge_static_libraries(cephd_compressor cephd_compressor_base
```

Comment:
could you take this opportunity to reformat the merged libraries? so we have one of them each line.